editor/src/components/canvas/canvas-strategies/strategies/keyboard-absolute-move-resize.spec.browser2.tsx

```diff
@@ -256,6 +264,117 @@ describe('Keyboard Absolute Resize E2E', () => {
       height: 101,
     })
   })
+
+  it('keeps trueuing up groups as directions change', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      formatTestProjectCode(
+        makeTestProjectCodeWithSnippet(`
+          <div data-uid='root'>
+            <Group
+              style={{
+                position: 'absolute',
+                left: 0,
+                top: 0,
+                width: 150,
+                height: 250,
+                backgroundColor: 'white',
+              }}
+              data-uid='group'
+            >
+              <div
+                style={{
+                  height: '100%',
+                  position: 'absolute',
+                  left: 0,
+                  top: 0,
+                  width: 83,
+                  backgroundColor: 'blue',
+                }}
+                data-uid='left'
+              />
+              <div
+                style={{
+                  backgroundColor: 'red',
+                  position: 'absolute',
+                  left: 100,
+                  top: 26,
+                  width: 50,
+                  height: 16,
+                }}
+                data-uid='right'
+              />
+            </Group>
+          </div>
+        `),
+      ),
+      'await-first-dom-report',
+    )
+
+    const groupPath = EP.fromString(
+      `${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:root/group`,
+    )
+    await renderResult.dispatch([selectComponents([groupPath], false)], true)
+
+    await keyDownArrowLeftHoldingCmd()
+    await keyDownArrowLeftHoldingCmd()
+    await keyDownArrowLeftHoldingCmd()
+    await keyDownArrowLeftHoldingCmd()
+    await keyDownArrowLeftHoldingCmd()
+    await keyDownArrowLeftHoldingCmd()
+
+    await keyDownArrowDownHoldingCmd()
+    await keyDownArrowDownHoldingCmd()
+    await keyDownArrowDownHoldingCmd()
+    await keyDownArrowDownHoldingCmd()
+    await keyDownArrowDownHoldingCmd()
+
+    clock.current.tick(KeyboardInteractionTimeout)
+
+    await wait(500)
```

Comment:
Is this still needed with the line above?

Reply:
yup!